src/sentry/models/auditlogentry.py

```diff
@@ -71,6 +71,10 @@ class AuditLogEntryEvent(object):
     SERVICEHOOK_ENABLE = 103
     SERVICEHOOK_DISABLE = 104
 
+    INTEGRATION_ADD = 105
```

Comment:
Move these down to starting at `110` just to leave room for others between.

Reply:
updated!